standard/requirements/coverage-scaling/REQ_cov-scaling-definition.adoc

```diff
@@ -0,0 +1,31 @@
+[[req_coverage_scaling-definition]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/coverage-scaling/definition*
+^|A |The operation SHALL support a parameter `scaleSize` with the following characteristics (using an Backus Naur Form (BNF) fragment):
+
+[source,BNF]
+----
+  ScalingSpec:       scaleSize=axisName({number})[,axisName({nuber})]*
+  axisName:          {NCName}
+
+  Where:
+     {NCName} is an XML-style identifier not containing ":" (colon) characters,
```

Comment:
Why limit to NCName and why mention XML?

Reply:
I copied this from the subset requirements which mentioned this.
That requirement not to have a colon might come from WCS?

Reply:
> I copied this from the subset requirements which mentioned this.
> That requirement not to have a colon in the axis name might come from WCS?

Indeed, this is relevant specifically for XML encodings, dating back to before coverages emancipated from GML. In CIS 1.1 such restrictions consequently hold when using the particular encodings, so in this case would constrain only XML-encoded (maybe also RDF?) requests. Wouldn't make sense for JSON indeed.